lib/pageflow/revision_components.rb

```diff
@@ -0,0 +1,7 @@
+module Pageflow
+  class RevisionComponents < Array
```

Comment:
Inheriting from `Array` makes the collection mutable. Removing a revision component after the app has booted is not supported, though. Some of the other collection classes in this directory include `Enumerable` instead.